guide/results-viewer-react/src/components/YamlUrls/index.tsx

```diff
@@ -137,65 +143,25 @@ export function Urls ({ data: { headers, ...data }, ...props }: UrlProps) {
     }
   }, [props.defaultHeaders]);
 
-  // Currently unoperational. Needs a server to access websites correctly.
-  // Would recommend simply using Postman instead of getting this function to work.
-  // Postman is already used by a lot of the FamilySearch org and does what this function below would do and more.
-  // Users can also simply create the testing script, test endpoints locally with pew pew, and edit the endpoints as needed.
-  const checkUrl = () => {
-    alert("Currently not working. Sorry!");
-    // console.log(data.url);
-    // const myUrl = `https://cors-escape-git-master.shalvah.now.sh/${data.url}`;
-    // console.log(myUrl);
-    /* let xhr = new XMLHttpRequest();
-    xhr.onreadystatechange = function () {
-      if (readyState === 4 && status === 200) {
-        let response = responseText;
-        console.log(response);
-      }
+  async function makeRequest () {
+    const request: AxiosRequestConfig = {
+      method,
+      url,
+      headers: headersList.reduce((acc, header) => {
+        acc[header.name] = header.value;
+        return acc;
+      }, {} as Record<string, string>),
+      validateStatus: () => true
+    };
+    try {
+      const response = await axios(request);
+      setLastResponseCode(response.status + " " + response.statusText);
+      setLastResponse(response.data);
+      return response.data;
+    } catch (error) {
+      throw error;
```

Comment:
Throwing the error are we displaying the error to the user somehow? Locally I'm having to look at the console to see the CORS errors I was getting.

Reply:
`(error as AxiosError).isAxiosError` will tell you if it's an `AxiosError` type.